tools/BaseRunner.py

```diff
@@ -34,3 +35,11 @@ def run(self, tmp_dir, params):
         log, _ = proc.communicate()
 
         return (log.decode('utf-8'), proc.returncode)
+
+##
+# This method is called by the main runner script (tools/runner) as a sanity
+# check to verify that tool is properly installed.
+# If False is returned then the test is skipped.
+# @return True when tool is installed and can be used, False otherwise.
```

Comment:
Why are the comments not a proper python doc strings?

Please convert the comments here (and everywhere) using the proper conventions (see https://www.python.org/dev/peps/pep-0257/ )